CheckDependenciesIT.testNoMandatoryDependencyDeclared

*sonar*
```
java.lang.AssertionError: Cannot find Hazelcast manifest
	at com.hazelcast.osgi.CheckDependenciesIT.findHazelcastManifestURL(CheckDependenciesIT.java:108)
	at com.hazelcast.osgi.CheckDependenciesIT.getHazelcastManifest(CheckDependenciesIT.java:61)
	at com.hazelcast.osgi.CheckDependenciesIT.testNoMandatoryDependencyDeclared(CheckDependenciesIT.java:48)
```

https://hazelcast-l337.ci.cloudbees.com/view/Sonar/job/Hazelcast-3.x-sonar/com.hazelcast$hazelcast/1592/testReport/junit/com.hazelcast.osgi/CheckDependenciesIT/testNoMandatoryDependencyDeclared/

Also failed locally:
```java
java.lang.AssertionError: Cannot find Hazelcast manifest

	at com.hazelcast.osgi.CheckDependenciesIT.findHazelcastManifestURL(CheckDependenciesIT.java:108)
	at com.hazelcast.osgi.CheckDependenciesIT.getHazelcastManifest(CheckDependenciesIT.java:61)
	at com.hazelcast.osgi.CheckDependenciesIT.testNoMandatoryDependencyDeclared(CheckDependenciesIT.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:105)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:97)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

Also failed locally on the client test:
**ClientCheckDependenciesIT.testNoMandatoryDependencyDeclared()**
```java
java.lang.AssertionError: Cannot find Hazelcast manifest

	at com.hazelcast.osgi.CheckDependenciesIT.findHazelcastManifestURL(CheckDependenciesIT.java:108)
	at com.hazelcast.osgi.CheckDependenciesIT.getHazelcastManifest(CheckDependenciesIT.java:61)
	at com.hazelcast.osgi.CheckDependenciesIT.testNoMandatoryDependencyDeclared(CheckDependenciesIT.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:105)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:97)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/hazelcast-all/src/test/java/com/hazelcast/it/CheckAllDependenciesIT.java
+++ b/hazelcast-all/src/test/java/com/hazelcast/it/CheckAllDependenciesIT.java
@@ -6,6 +6,6 @@ public class CheckAllDependenciesIT extends CheckDependenciesIT {
 
     @Override
     protected boolean isMatching(String urlString) {
-        return urlString.contains("hazelcast/hazelcast/") && !urlString.contains("test");
+        return urlString.contains("hazelcast-all-") && urlString.contains("target");
     }
 }

--- a/hazelcast-client/src/test/java/com/hazelcast/client/osgi/ClientCheckDependenciesIT.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/osgi/ClientCheckDependenciesIT.java
@@ -6,6 +6,6 @@ public class ClientCheckDependenciesIT extends CheckDependenciesIT {
 
     @Override
     protected boolean isMatching(String urlString) {
-        return urlString.contains("hazelcast-client/target/classes/META-INF/MANIFEST.MF");
+        return urlString.contains("hazelcast-client-") && urlString.contains("target");
     }
 }

--- a/hazelcast/src/test/java/com/hazelcast/osgi/CheckDependenciesIT.java
+++ b/hazelcast/src/test/java/com/hazelcast/osgi/CheckDependenciesIT.java
@@ -8,7 +8,6 @@ import com.hazelcast.test.HazelcastTestSupport;
 import com.hazelcast.test.annotation.QuickTest;
 import org.apache.felix.utils.manifest.Clause;
 import org.apache.felix.utils.manifest.Parser;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
@@ -23,8 +22,9 @@ import static org.junit.Assert.fail;
 
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
-@Ignore
 public class CheckDependenciesIT extends HazelcastTestSupport {
+
+
     private static final String MANIFEST_PATH = "META-INF/MANIFEST.MF";
     private static final String[] WHITELIST_PREFIXES = new String[]{
 
@@ -117,6 +117,6 @@ public class CheckDependenciesIT extends HazelcastTestSupport {
     }
 
     protected boolean isMatching(String urlString) {
-        return urlString.contains("hazelcast/target/classes/META-INF/MANIFEST.MF");
+        return urlString.contains("hazelcast-3.") && urlString.contains("target");
     }
 }

--- a/pom.xml
+++ b/pom.xml
@@ -591,6 +591,9 @@
                             <includes>
                                 <include>**/*.java</include>
                             </includes>
+                            <excludes>
+                                <exclude>**/**IT.java</exclude>
+                            </excludes>
                             <groups>
                                 com.hazelcast.test.annotation.QuickTest,
                                 com.hazelcast.test.annotation.SlowTest
@@ -987,6 +990,7 @@
                                 <include>**/**.java</include>
                             </includes>
                             <excludes>
+                                <exclude>**/**IT.java</exclude>
                                 <exclude>**/jsr/**.java</exclude>
                             </excludes>
                             <groups>

